pkg/util/utils_test.go

```diff
@@ -43,6 +43,42 @@ func TestGetInput(t *testing.T) {
 	assert.EqualValues(input, result)
 	_ = restoreOutput()
 
+	// GetInput should remove single quotes from start and end
```

Comment:
No need to change, but this would be better as an array of values and expectations and a for loop.